packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/info-summary.ts

```diff
@@ -0,0 +1,31 @@
+import { InfoElement } from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement } from '@swagger-api/apidom-core';
+
+type InfoSummaryPluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const infoSummaryRefractorPlugin =
+  ({ annotations }: InfoSummaryPluginOptions) =>
+  () => {
+    const annotation = new AnnotationElement(
+      'The "summary" field of Info Object is not supported in OpenAPI 3.0.3. It will be removed from the converted document.',
```

Comment:
We're using past tense for annotations, describing things that already happened.

```suggestion
      'The "summary" field of Info Object is not supported in OpenAPI 3.0.3. It has been removed from the converted document.',
```